### PR TITLE
test: use different vendor settings for styling pages

### PR DESCRIPTION
**Pull request by danielgent**

### CODE REVIEW

File: src/types/sql/Tenant.ts [hunk 1]
```diff
@@ -40,9 +40,23 @@ export type VendorSettings = {
...
+    selfHandlesVat: boolean;
+    invoiceAddress: {
+      country: string;
+      zipCode: string;
+      city: string;
+      street: string;
+    };
+    taxId: string;
+  };
+  manageSubscriptionsUrl?: string;
```
Comment (danielgent) [inline src/types/sql/Tenant.ts:61]: I manually copy-pasted real vendor settings into our fixtures and this matches what we're actually getting back

I could remove the keys we're not using

File: test/integration/flows/code-flow.spec.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { tsoaApp, loginApp } from "../../../src/app";
 import { getAdminToken } from "../helpers/token";
 import { getEnv } from "../helpers/test-client";
 import { EmailOptions } from "../../../src/services/email/EmailOptions";
-import { snapshotEmail } from "../helpers/snapshotEmailsPlaywright";
+import { snapshotEmail } from "../helpers/playwrightSnapshots";
```
Comment (danielgent) [inline test/integration/flows/code-flow.spec.ts:11]: I colocated this helper with the other

File: test/integration/flows/password.spec.ts [hunk 3]
```diff
@@ -825,7 +832,9 @@ describe("password-flow", () => {
   });
   describe("Password reset", () => {
     it("should send password reset email for existing user, and allow password to be changed", async () => {
-      const env = await getEnv();
+      const env = await getEnv({
+        vendorSettings: FOKUS_VENDOR_SETTINGS,
```
Comment (danielgent) [inline test/integration/flows/password.spec.ts:836]: this is the biggest change in this PR. Seems a logical way to seed the vendor settings

File: test/integration/helpers/test-client.ts [hunk 4]
```diff
@@ -14,8 +14,15 @@ import { mockOAuth2ClientFactory } from "../mockOauth2Client";
...
 import type { Client } from "../../../src/types";
 import type { EmailOptions } from "../../../src/services/email/EmailOptions";
+import { SESAMY_VENDOR_SETTINGS } from "../../fixtures/vendorSettings";
+
+type getEnvParams = {
+  vendorSettings?: VendorSettings;
+};
+
+export async function getEnv(args?: getEnvParams) {
+  const vendorSettings = args?.vendorSettings ?? SESAMY_VENDOR_SETTINGS;
```
Comment (danielgent) [inline test/integration/helpers/test-client.ts:24]: I always hate that this can't be done straight in the function signature :smiling_face_with_tear:

**Review by markusahlstrand — APPROVED** [review]
LGTM

**Pull request merged**